src/elements/a-progressive-asset.js

```diff
@@ -0,0 +1,57 @@
+AFRAME.registerElement("a-progressive-asset", {
+  prototype: Object.create(AFRAME.ANode.prototype, {
+    createdCallback: {
+      value() {
+        this.data = null;
+        this.isAssetItem = true;
+        this.fileLoader = document.querySelector("a-assets").fileLoader;
```

Comment:
Hmm, I don't think there is anything special about a-asset's fileloader so we should maybe just create our own. If not, we should probably get it from this element's parent node (since its required to be in an a-assets block), as there could be multiple scenes in the document.

Reply:
Gonna go with the later and reuse the FileLoader from the parent.